### ISSUE TITLE
bug with DefaultFormatBundle in test_pipeline when running image_demo.py

### ISSUE THREAD

**Issue by hyz-xmaster** [open-mmlab/mmdetection#4091]
Thanks for your error report and we appreciate it a lot.

**Checklist**
1. I have searched related issues but cannot get the expected help.
2. The bug has not been fixed in the latest version.

**Describe the bug**
See here [run demo](https://github.com/hyz-xmaster/VarifocalNet/issues/6). I think this bug is caused by changing `ImageToTensor` to `DefaultFormatBundle` in the config file. Something wrong with `DefaultFormatBundle` in test_pipeline. The actual bug happens in [mmcv](https://github.com/open-mmlab/mmcv/blob/d9ef9dabe2bbe9950e33c21fec36c94bcab622b2/mmcv/parallel/scatter_gather.py#L27) and [here](https://github.com/open-mmlab/mmcv/blob/d9ef9dabe2bbe9950e33c21fec36c94bcab622b2/mmcv/parallel/_functions.py#L72)

**Reproduction**
1. What command or script did you run?
```
 python demo/image_demo.py demo/demo.jpg configs/vfnet/vfnet_r50_fpn_1x_coco.py checkpoints/vfnet_r50_1x_41.6.pth
```
2. Did you make any modifications on the code or config? Did you understand what you have modified?

   Change `ImageToTensor` to `DefaultFormatBundle` in the config file.

3. What dataset did you use?

   COCO

**Environment**

1. Please run `python mmdet/utils/collect_env.py` to collect necessary environment information and paste it here.

   mmcv = 1.1.5 pytorch = 1.6.0 mmdet=2.6.0

**Error traceback**

see [here](https://github.com/hyz-xmaster/VarifocalNet/issues/6)

**Bug fix**

Something wrong with `DefaultFormatBundle` in test_pipeline. Changing it back to `ImageToTensor` solves the problem.

**Comment by ZwwWayne** [comment]
Hi @hyz-xmaster,
Thanks for your bug report. Would you like to create a PR to fix that?

**Comment by hyz-xmaster** [comment]
Just back from holiday. It seems this bug has been fixed by #4381.
